src/Features/JsonPatch.SystemTextJson/src/Operations/Operation.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.JsonPatch.SystemTextJson.Adapters;
+using Microsoft.AspNetCore.JsonPatch.SystemTextJson;
+using Microsoft.AspNetCore.Shared;
+
+namespace Microsoft.AspNetCore.JsonPatch.SystemTextJson.Operations;
+
+public class Operation : OperationBase
+{
+    [JsonPropertyName(nameof(value))]
+    public object value { get; set; }
+
+    public Operation()
+    {
+    }
+
+    public Operation(string op, string path, string from, object value)
+        : base(op, path, from)
+    {
+        this.value = value;
+    }
+
+    public Operation(string op, string path, string from)
+        : base(op, path, from)
+    {
+    }
+
+    public void Apply(object objectToApplyTo, IObjectAdapter adapter)
+    {
+        ArgumentNullThrowHelper.ThrowIfNull(objectToApplyTo);
+        ArgumentNullThrowHelper.ThrowIfNull(adapter);
+
+        switch (OperationType)
+        {
+            case OperationType.Add:
+                adapter.Add(this, objectToApplyTo);
+                break;
+            case OperationType.Remove:
+                adapter.Remove(this, objectToApplyTo);
+                break;
+            case OperationType.Replace:
+                adapter.Replace(this, objectToApplyTo);
+                break;
+            case OperationType.Move:
+                adapter.Move(this, objectToApplyTo);
+                break;
+            case OperationType.Copy:
+                adapter.Copy(this, objectToApplyTo);
+                break;
+            case OperationType.Test:
+                if (adapter is IObjectAdapterWithTest adapterWithTest)
+                {
+                    adapterWithTest.Test(this, objectToApplyTo);
+                    break;
+                }
+                else
+                {
+                    throw new NotSupportedException(Resources.TestOperationNotSupported);
+                }
+            default:
+                break;
+        }
+    }
+
+    public bool ShouldSerializevalue()
```

Comment:
```suggestion
    public bool ShouldSerializeValue()
```

Reply:
I remember this one, I decided to keep this as is, as that API had the same casing in the original codebase: https://github.com/dotnet/aspnetcore/blob/c22a8530ee463bf3534ce5fc54f991e8ab1e9ee0/src/Features/JsonPatch/src/Operations/Operation.cs#L68

But now that I think about this, this is the chance to get this fixed, so I'll update it! Thanks!

Reply:
Or it looks like I missed this one as part of my last commit, where I removed a few other unused methods. Will remove it.